### PR TITLE
OKTA-225387-Update samples-ios to use okta-auth-swift version 1

### PR DESCRIPTION
**Pull request by IldarAbdullin-okta**
- Implemented flow coordinator and flow coordinator delegate
- View controllers are lightweight

### CODE REVIEW

File: custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    var oidcStateManager: OktaOidcStateManager?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        successStatus = status as? OktaAuthStatusSuccess
+        titleLabel.text = "Welcome, \(successStatus?.model.embedded?.user?.profile?.firstName ?? "-")"
+        subtitleLabel.text = successStatus?.model.embedded?.user?.profile?.login
+        timezoneLabel.text = successStatus?.model.embedded?.user?.profile?.timeZone
+        localeLabel.text = successStatus?.model.embedded?.user?.profile?.locale
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```
Comment (anastasiiaiurok-okta) [inline custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift]: Why is this needed? Maybe move this part to viewDidAppear?

File: custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                        self.accessTokenLabel.textColor = UIColor(red: 0, green: 255, blue: 0, alpha: 1)
+                    }
+                    if let _ = stateManager?.refreshToken {
+                        self.refreshTokenLabel.text = "YES"
+                        self.refreshTokenLabel.textColor = UIColor(red: 0, green: 255, blue: 0, alpha: 1)
+                    }
+                    self.oidcStateManager = stateManager
+                })
+            } else {
+                SVProgressHUD.dismiss()
```
Comment (anastasiiaiurok-okta) [inline custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift]: refactoring this to `guard let ` construction will make code more readable.

Comment (IldarAbdullin-okta) [reply custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift]: Agree

File: custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift (hunk 1 above)
Comment (IldarAbdullin-okta) [reply custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift]: `viewDidAppear` can be called by iOS multiple times. I need to run this code exactly once
One second delay is intentional, I want to show basic information on the screen and then request access token
